### ISSUE TITLE
CLI 'develop' on fresh project spams weird text infinitely

### ISSUE THREAD

**Issue by tennox** [strapi/strapi#12142]
## Bug report
I just started a new project and now strap is spamming weird stuff at me :sweat_smile: 

https://user-images.githubusercontent.com/2084639/148699056-16b133cf-0330-43ae-9730-e9fbfaee46b2.mp4

### Describe the bug

~~A clear and concise description of what the bug is.~~
You tell me :see_no_evil: 

### Steps to reproduce the behavior

0. (optional) open fresh environment `docker run --rm -it node:16 bash`
1. `npx create-strapi-app@latest my-project --quickstart`
2. See spam after some time
3. go into my-project folder
4. run `yarn develop`
5. See spam

### Screenshot
![Screenshot from 2022-01-09 21-27-46](https://user-images.githubusercontent.com/2084639/148699560-442d3356-af2b-4b47-922a-c426dc547c74.png)

### System

- Node.js version: 16
- NPM version: 6.14.15
- Strapi version: 4.0.3
- Operating system: Ubuntu

**Comment by lItc0de** [comment]
I have the same problem

**Comment by mass22** [comment]
Same here!

**Comment by tennox** [comment]
Found the issue... It's the `colors` npm package... 
NPM FTW :sunglasses:  :tada: 

https://daytonews.com/2022/01/09/dev-corrupts-npm-libs-colors-and-faker-breaking-thousands-of-apps/

**Comment by nikolajbech** [comment]
Add this to your package.json:

`"resolutions": {
    "colors": "1.2.1"
  }`

**Comment by TheSamDickey** [comment]
> Add this to your package.json:
> 
> `"resolutions": { "colors": "1.2.1" }`

This did not solve the problem for me

**Comment by mariansimecek** [comment]
Same problem, installing older version of colors solved problem! 
`npm install colors@1.2.1`

**Comment by jerichosiahaya** [comment]
> Same problem, installing older version of colors solved problem! `npm install colors@1.2.1`

This actually solve the problem for me.

**Comment by paul-corbett-qantas** [comment]
Issue deliberately caused by the owner of `https://github.com/Marak/colors.js` 

Fixed temporarily by adding
```
"resolutions": {
    "colors": "1.3.3"
  }
```
 to package.json and running `yarn install`

**Comment by corbpaul** [comment]
Issue deliberately caused by the owner of `https://github.com/Marak/colors.js` 

Fixed temporarily by adding
```
"resolutions": {
    "colors": "1.3.3"
  }
```
 to package.json and running `yarn install`

**Comment by EzraEn1** [comment]
According to the dependency tree, the colors.js injection appears to be coming from `cli-table3v0.6.1`. Will be issuing a PR to try and resolve this permanently.

**Comment by EzraEn1** [comment]
Fixable by updating cli-table3 to 0.6.1. Issued PR to do just that.

**Comment by anildalar** [comment]
same problem

**Comment by eltharynd** [comment]
same problem

**Comment by EzraEn1** [comment]
@eltharynd, a hotfix is being prepped, hold on for a permanent fix! 😁

**Comment by eltharynd** [comment]
Thanks.

I can confirm that running
`npm uninstall cli-table3`
and
`npm install cli-table3@0.6.1`
temporarily fixes it

**Comment by derrickmehaffy** [comment]
Fixed in v4.0.4
